examples/ftd_api_scripts/timezone.py

```diff
@@ -0,0 +1,96 @@
+'''
+Copyright (c) 2019 Cisco and/or its affiliates.
+This software is licensed to you under the terms of the Cisco Sample
+Code License, Version 1.1 (the "License"). A copy of the License
+can be found in the LICENSE.TXT file of this software or at
+https://developer.cisco.com/site/license/cisco-sample-code-license/
+All use of the material herein must be in accordance with the terms of
+the License. All rights not expressly granted by the License are
+reserved. Unless required by applicable law or agreed to separately in
+writing, software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+express or implied.
+'''
+
+import sys
+from ftd_api_resources.timezone_utils import get_all_time_zones, post_time_zone, update_time_zone, get_time_zone, \
+    delete_time_zone
+from ftd_api_resources.access_token import get_access_token
+
+
```

Comment:
Please make these corrections:
time_zone_actions() should return True or False
Add params list to docstring, remove 'you must update...' line
main block should exit(0) or exit(1)

Reply:
Done in https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/8